src_assets/common/assets/web/index.html

```diff
@@ -107,21 +107,27 @@ <h2>Legal</h2>
         let v = this.githubVersion.name;
         // If the version starts with a v, remove it
         if (v.indexOf("v") === 0) v = v.substring(1);
+
+        // if nightly, we do an additional check to make sure its an actual upgrade.
+        if (this.version.split('.').length > 3) {
+          const stableVersion = this.version.split('.').slice(0, 3).join('.');
+          return this.githubVersion.tag_name.substring(1) > stableVersion;
+        }
+
         // return true if the version is different, otherwise false
         return v !== this.version.split(".")[0];
       },
-      // Check for new nightly version
       hasNewNightly() {
         // If we're not on a nightly build, just return false
         // If length of version split is 3, we're on a stable build
         if (!this.version || !this.nightlyData || this.version.split(".").length === 3) return false;
         // If built with dirty git tree, return false
         if (this.version.indexOf("dirty") !== -1) return false;
         // Get the commit hash
-        let commit = this.version.split(".")[-1];
+        let commit = this.version.split(".").pop();
         // return true if the commit hash is different, otherwise false
         return this.nightlyData.head_sha.indexOf(commit) !== 0;
       }
     }
   });
-</script>
+</script>
```

Comment:
Please undo this (re-add blank line at end of file).

Reply:
Not sure what you mean, I had to add code statements and that shifted the script tag down a few lines?

Reply:
The line number changing isn't the issue. Your IDE might be screwing with it. There should be a blank line at the end of each file though. You can see this on the files changed tab. https://github.com/LizardByte/Sunshine/pull/1073/files

![image](https://user-images.githubusercontent.com/42013603/227731217-c4407a86-fcf5-4867-a210-43f13f157518.png)